Config/Pipeline/configdefaults201012.yaml

```diff
@@ -30,32 +30,33 @@ java:
 singularity:
     default: "docker://gmsuppsala/somatic:develop"
     execute: "singularity exec -B /data -B /projects -B /scratch "
-    multiqc: "/projects/wp4/nobackup/workspace/somatic_dev/singularity/multiqc-1.9.simg"
-    samtools: "/projects/wp2/nobackup/Twist_Myeloid/Containers/bwa0.7.17-samtools-1.9.simg"
+    #multiqc: "/projects/wp4/nobackup/workspace/somatic_dev/singularity/multiqc-1.9.simg"
+    samtools_fgbio: "/projects/wp2/nobackup/Twist_Myeloid/Containers/bwa0.7.17-samtools-1.9.simg"
     #picard: "/projects/wp2/nobackup/Twist_Myeloid/Containers/bwa0.7.17-samtools-1.9.simg"
-    bwa: "/projects/wp2/nobackup/Twist_Myeloid/Containers/bwa0.7.17-samtools-1.9.simg"
+    bwa_fgbio: "/projects/wp2/nobackup/Twist_Myeloid/Containers/bwa0.7.17-samtools-1.9.simg"
```

Comment:
Why do the regular bwa version not work?

Reply:
It does but the fgbio rule needs to be split up into several steps so I kept the old singularity for now

Reply:
ok